blueprints/serverless/cloud-run-microservices/alb.tf

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Internal Application Load Balancer resource.
+
+# Internal Application Load Balancer in main (host) project
+module "int-alb" {
+  source     = "../../../modules/net-lb-app-int"
+  count      = var.prj_svc1_id != null ? 1 : 0
```

Comment:
I know this follows what is described in `README.md` but results in a bit of magic behavior

Reply:
I'm following this pattern in these cloud run blueprints because it allows to create several related use cases inside a blueprint simply (it's not so _magic_), reusing 80-90% of code instead of creating multiple very similar blueprints.

Reply:
Totally agree, that there is no need for separate blueprints to cover very similar cases. What I don't like about it, that based on whether service project id is provided or not, so many things do change. But I guess there is no other simple way, and I consider simplicity to be a top-priority here.